### ISSUE TITLE
Backslash on table name breaks it

### ISSUE THREAD

**Issue by TCB13** [fpdo/fluentpdo#106]
Hello,

I was trying to get records out of a table called `CMS\Page`, yes there's a backslash on the table name. Unfortunately the backslash seems to result in a strange JOIN behavior.

**Query without backslash:**

````
$fpdo = new \FluentPDO($pdo);

$query = $fpdo->from("Page")
->select(array('title', 'date'))
->limit(5);

print $query->getQuery();
````

=> `SELECT Page.*, title FROM Page LIMIT 5`

**Query with backslash on table name:**

````
$fpdo = new \FluentPDO($pdo);

$query = $fpdo->from("CMS\\Page")
->select(array('title', 'date'))
->limit(5);

print $query->getQuery();
````

=> `SELECT CMS\Page.*, title, date FROM CMS\Page LEFT JOIN Page ON Page.id = CMS\Page.Page_id LIMIT 5`

Why do this happen? There's no reason why we shouldn't have backslashed on table names. At least not on the most recent MySQL/MariaDB versions.

**Side Note:** There's also the old annoying issue where using `->select(...)` doesn't matter because `->from(...)` will always add a select all on the table like `Page.*`.

Thank you.

**Comment by cbornhoft** [comment]
For your second issue, use `$fpdo->from()->select(null)->select('column1','column2');`

This is because the smart join builder thinks you're performing a join with select.

**Comment by maldoinc** [comment]
Hello, thanks for taking the time to report this. Escaping table and column names is a common request but for this current release we're not automatically escaping them due to backwards compatibility.

**Comment by cbornhoft** [comment]
Fixed in #209
